### ISSUE TITLE
[DOCS] Fixing Deployment with DocumenterVitepress

### ISSUE THREAD

**Issue by TheCedarPrince** [JuliaHealth/OMOPCDMPathways.jl#15]
Hey @Jay-sanjay, 

I was doing some review and I believe the way to fix this issue is to take this file:

https://github.com/JuliaHealth/OMOPCDMCohortCreator.jl/blob/f78c77e782c5ca8e74a32a2113a27a9227c09018/docs/make.jl

and reconcile it with the DocumenterVitepress implementation you made here:

https://github.com/JuliaHealth/OMOPCDMPathways/blob/cabfbdd97588acaa8f6b730ae10d9058e76f2b68/docs/make.jl

In particular, this piece from OCC:

```julia
makedocs(;
    modules = [OMOPCDMCohortCreator],
    authors = "Jacob Zelko (aka TheCedarPrince) <jacobszelko@gmail.com> and contributors",
    repo = "https://github.com/JuliaHealth/OMOPCDMCohortCreator.jl/blob/{commit}{path}#L{line}",
    sitename = "OMOPCDMCohortCreator.jl",
    format = Documenter.HTML(;
        prettyurls = get(ENV, "CI", "false") == "true",
        canonical = "https://JuliaHealth.github.io/OMOPCDMCohortCreator.jl",
        assets = String[],
        edit_link = "dev",
	footer = "Created by [Jacob Zelko](https://jacobzelko.com) & [Georgia Tech Research Institute](https://www.gtri.gatech.edu). [License](https://github.com/JuliaHealth/OMOPCDMCohortCreator.jl/blob/main/LICENSE)"
    ),
    pages = [
        "Home" => "index.md",
        "Tutorials" => [
                        "tutorials.md",
                        "tutorials/beginner_tutorial.md",
                        "tutorials/r_tutorial.md"
                       ],
        "API" => "api.md",
        "Contributing" => "contributing.md"
    ],
)
```

Needs to be combined with:

```julia
fmt  = DocumenterVitepress.MarkdownVitepress(
    repo="https://github.com/JuliaHealth/OMOPCDMPathways",
    devurl = "dev",
    # deploy_url = "yourgithubusername.github.io/OMOPCDMPathways.jl",
    build_vitepress = false,
)
# )

makedocs(;
    modules = [OMOPCDMPathways],
    authors = "Jay-sanjay <landgejay124@gmail.com> and contributors",
    repo = "https://github.com/JuliaHealth/OMOPCDMPathways",
    sitename = "OMOPCDMPathways.jl",
    format = fmt,
    pages= pgs,
    warnonly = true,
)
```

I just don't know how format works for DocumenterVitepress. You may want to ask in the Julia `#documentation` Slack channel for help or comment over in [DocumenterVitepress](https://github.com/LuxDL/DocumenterVitepress.jl) for some guidance. Happy to brainstorm on this some more as needed!

~ tcp :deciduous_tree:

**Comment by TheCedarPrince** [comment]
Also, looking at this repo, it seems like you didn't generate a [`Documenter` deploy key or a `DOCUMENTER_KEY` GitHub secrets ](https://documenter.juliadocs.org/stable/man/hosting/#Authentication:-SSH-Deploy-Keys) like in OCC (you can see this stuff in the settings of that repository -- let me know if you cannot see it).

**Comment by Jay-sanjay** [comment]
Is this correct way for it?
https://github.com/JuliaHealth/OMOPCDMPathways.jl/blob/5c8cb3d60a100a416fc120cf20809c30ea267a0e/.github/workflows/documentation.yml#L48

**Comment by TheCedarPrince** [comment]
Yup! And did you generate the keys in the settings of the repo @Jay-sanjay ?

**Comment by TheCedarPrince** [comment]
Also, did you update the make.jl file like I was talking about in the comment?

**Comment by Jay-sanjay** [comment]
> Yup! And did you generate the keys in the settings of the repo @Jay-sanjay ?

I did not get that setting

**Comment by TheCedarPrince** [comment]
Ah ok -- let's work on it tomorrow during our meeting. :)

**Comment by Jay-sanjay** [comment]
Fixed using [#14](https://github.com/JuliaHealth/OMOPCDMPathways.jl/pull/19)
